### DISCUSSION TITLE
Whats the difference between FreeMcBoot 1.966, 1.965 and 1.953?

### DISCUSSION THREAD

**Discussion by matcarfer** [israpps/FreeMcBoot-Installer#18] [Announcements]
Thats all. Maybe the answer to my question should be on the README.md

**Comment by israpps** [comment]
> Thats all. Maybe the answer to my question should be on the README.md

The differences can already be seen on the [changelog bundled to every package](https://github.com/israpps/FreeMcBoot-Installer/blob/master/1966/__Package/Changelog/Changelog.txt)

**Comment by israpps** [comment]
I hate seeing the number on the issue tab

Therefore, and due to the nature of this question

I'm converting this into a discussion

**Comment by matcarfer** [comment]
Ok thanks for answering. Now I wonder, isnt the latest the best release? Why keep the older ones?

**Reply by israpps** [reply]
because latest release has memory conflicts with modchips

and we can't fix them.
Even if the code was released...

**Reply by matcarfer** [reply]
wow i didnt know this, and my 2 ps2s slims have modchips. how can I identify which version is correct?

**Reply by israpps** [reply]
simple.

conflictive versions won't work or won't execute any Application

**Reply by matcarfer** [reply]
okey, will test to check out, this info is new to me, sorry for bothering and thanks for the help!

**Reply by Helxgamer** [reply]
I asked exactly that a few days ago , version 1966 really didn't work, the curious thing is that if you reset the console after loading it once , it starts working normally , I don't know if the information is wrong is useful but the comment is here , thanks to developer for keeping this project alive, any problem using the 1965 version that works normally, not that the latest one doesn't, it's our fault for having modship.

**Comment by spankm** [comment]
Does this support hard drives formatted as FAT?

**Reply by israpps** [reply]
> Does this support hard drives formatted as FAT?

u mean EXFAT?

no, and never will. FreeMcBoot/FreeHdBoot is closed source
